hasher-matcher-actioner/hmalib/common/writebacker_models.py

```diff
@@ -151,6 +151,23 @@ def te_api(self) -> ThreatExchangeAPI:
         api_key = AWSSecrets().te_api_key()
         return ThreatExchangeAPI(api_key)
 
+    def my_descriptor(
+        self, descriptors: t.List[t.Dict[str, t.Any]]
+    ) -> t.Optional[t.Dict[str, t.Any]]:
+        print(descriptors)
+        for descriptor in descriptors:
```

Comment:
suggest adding a comment or docstring like:
`loop until we find the one descriptor we own for a given indicator (if it exists)`